requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
+pytest
+pytest-cov
+pytest-pep8
```

Comment:
I'm fine with either, too.  I guess the few extra requirements are probably okay for people who only care about docs but not pytest.